app/src/docs/Components/Connect/ExtensionAccountsProvider/main.tsx

```diff
@@ -0,0 +1,164 @@
+      />
+      <p>
+        <a
+          href="https://github.com/paritytech/polkadot-cloud/blob/main/packages/cloud-react/lib/connect/ExtensionAccountsProvider/index.tsx"
+          target="_blank"
+          rel="noreferrer"
+        >
+          <code>ExtensionAccountsProvider</code>
+        </a>{" "}
+        provides a simple means of connecting to web3 extensions that implement
+        the <code>window.injectedWeb3</code> interface, and subscribes to their
+        accounts. It automatically re-connects to extensions on subsequent
+        visits, and allows dapps to set an active account if found in an
+        extension.
+      </p>
+
+      <Note>
+        <p>
+          This provider prevents the multiple pop-up issue (all available
+          extensions popping up when connecting to extensions) by design. Such
+          behaviour should be treated as an anti-pattern and be avoided.
+        </p>
+      </Note>
+
+      <Note>
+        <p>
+          EVM accounts are not currently supported by{" "}
+          <code>ExtensionAccountsProvider</code>.
+        </p>
+      </Note>
+
+      <h3>Requires:</h3>
+      <p />
+      <ul>
+        <li>
+          <Link to="/extensions_provider">
+            <code>ExtensionsProvider</code>
+          </Link>
+          : Must wrap <code>ExtensionAccountsProvider</code> to provide the
+          available extensions.
+        </li>
+      </ul>
+      <hr />
+
+      <h3>Providers Setup</h3>
+      <p>
+        Wrap your app with <code>ExtensionAccountsProvider</code> and provide
+        the required props. Note that <code>ExtensionsProvider</code> is also
+        required, in order to provide the available extensions to connect to.
+        The required props are referenced further down.
+      </p>
+
+      <ProviderExample />
+
+      <h3>Connecting Extensions</h3>
+
+      <p>
+        With the providers in place, you can call{" "}
+        <code>connectExtensionAccounts</code> to connect to an extension. The
+        following example attempts to find Subwallet JS from{" "}
+        <code>extensions</code>, and connects to it via a button if found:
+      </p>
+
+      <ConnectExample />
+
+      <p>
+        It is recommended to carry out validation checks, such as if the
+        extension is already connected and if it exists.
+      </p>
+
+      <h3>Getting Accounts</h3>
+
+      <p>
+        Once connected to an extension, the subscribed accounts become available
+        via <code>extensionAccounts</code>. The <code>address</code>,{" "}
+        <code>name</code> of the account, and <code>signer</code> are returned
+        for each account.
+      </p>
+
+      <AccountsExample />
+
+      <h2>Props</h2>
+
+      <h3 className="reference">dappName: string</h3>
+      <p>
+        A dapp identifier that is provided to the web3 extension(s) being
+        connected to.
+      </p>
+
+      <h3 className="reference">network: "polkadot" | "kusama" | "westend" </h3>
```

Comment:
```suggestion
      <h3 className="reference">network: "polkadot" | "kusama" | "westend" | "rococo" </h3>
```
Rococo should be supported